train.py

```diff
@@ -44,18 +44,7 @@
     from apex.parallel import convert_syncbn_model
     has_apex = True
 except ImportError:
-    if not hasattr(torch, 'moreh'):
```

Comment:
If we remove amp tests, then we don't need amp tests of NVIDIA GPU as well. 
It means we remove out all amp tests in our model config yamls?

Reply:
Ah I just saw https://github.com/moreh-dev/moreh-testbench/pull/197.